main.nf

```diff
@@ -6,6 +6,8 @@ params.csv_path = "data/input.csv"
 
 //checks metadata, and passes relavent fields along through .json
 process SYNAPSE_CHECK {
+
+    cache false
```

Comment:
I think this is the right approach. This way, the `SYNAPSE_CHECK` process will always run to check for new versions.